src/PokemonDetails.js

```diff
@@ -0,0 +1,10 @@
+import React, { Component } from 'react';
+
+class PokemonDetails extends Component {
+    state = {  }
+    render() { 
+        return (  <h2>Here comes the details</h2>);
```

Comment:
On this page you'll want it to be a bit like the PokeItem component in that it has information about a single pokemon, pulling from the params and from props passed down.